examples/semantic_3d.py

```diff
@@ -0,0 +1,42 @@
+"""
+Perform interactive semantic segmentation on 3D volume
+"""
+import numpy as np
+from napari import Viewer, gui_qt
+from skimage import data
+from skimage.color import rgb2gray
+from segmentify.semantic import fit, predict
+import napari
+import imageio
+import os
+print(napari.__version__)
+
+
+with gui_qt():
+
+    # create an empty viewer
+    viewer = Viewer()
+
+    # read in sample data
+    example_file = os.path.join(os.path.abspath(os.path.dirname(__file__)),"volume.npy")
```

Comment:
Can we name this something else like `image_stack` as when you say volume I think 3D volumetric data.

Also can you save the image data as a `.tif` file instead of `.npy` so it is language agnostic